interface/themes/style_light.css

```diff
@@ -50,17 +53,21 @@ a:hover {
 .text {
   text-decoration: none;
   color: #000000;
+  font-size: 1em;
 }
 
 .required {
   text-decoration: none;
   color: #880000;
   font-weight: bold;
+  font-size: 1em;
 }
 
 .bold {
   text-decoration: none;
+  color: #000000;
   font-weight: bold;
+  font-size: 1em;
```

Comment:
@robertdown ,

Having a tough time reconciling the above 3 classes with the css_button_small class.
The above 3 items are too large, especially because they are used in the form reports (vitals widget in patient summary, encounter, patient reports) and are breaking the 2nd column with a line break.

The issue is that if decrease this, then the Widget titles in patient summary get super small since they also get shrinked by the css_button_small class, it appears.

thoughts?

Otherwise these changes bring back relative font sizes from before wiping them all out and uses em for all font-sizes.